source/tutorials/learning-paths/pipelines/index.rst

```diff
@@ -0,0 +1,41 @@
+.. meta::
+    :description: Learn about building and running pipelines in Valohai.
+
+.. _learning-paths-pipelines:
+
+
+Valohai pipelines
+###################
+
+.. admonition:: Prerequirements
+    :class: attention
+
+    * :ref:`repository` before using pipelines.
```

Comment:
Same q here - Is Git a requirement or just strongly encouraged? Why not go with `--adhoc`?

Reply:
Same answer as well. :D